scss/pokestyles.scss

```diff
@@ -0,0 +1,86 @@
+$white: #ffffff;
+$mine-shaft-grey: #222222;
+
+$poke-red: #EF5350;
+$poke-blue: #198CD2;
+$poke-grey: #cccccc;
+$poke-yellow: #FECA1B;
+
+$logo-text-color: $poke-yellow;
+$logo-text-shadow: $mine-shaft-grey;
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: 'Cantarell', sans-serif;
+}
+
+@font-face {
+  font-family: Ketchum;
+  src: url('../fonts/Ketchum.otf');
+}
+
+header {
+  margin: 40px auto;
+  width: 400px;
+  height: 300px;
+  position: relative;
+}
+
+header h1 {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  margin: auto;
+  font-size: 4.2rem;
+  font-family: Ketchum;
+  color: $logo-text-color;
+  text-shadow: 1px 1px 2px $logo-text-shadow;
+}
+
+header img {
+  position: absolute;
+  transform: translate(-50%, -50%) rotate(-40deg) ;
+  left: 50%;
+  top: 50%;
+  height: 70%;
+}
+
+main {
+  margin: 0 auto;
+  width: 400px;
+}
+
+label {
```

Comment:
why? you have no label tags

Reply:
Removed the unused label styling in 2a810b2